test/src/test/java/org/corfudb/integration/CorfuReplicationMultiSourceSinkIT.java

```diff
@@ -165,6 +165,30 @@ protected void startReplicationServers() throws Exception {
         }
     }
 
+    protected void stopReplicationServer(int serverPort, Process serverProcess) {
+        List<String> paramsPs = Arrays.asList("/bin/sh", "-c", "ps aux | grep CorfuInterClusterReplicationServer | grep " + serverPort);
```

Comment:
Not sure I understand, you mean executing ITs/tests in parallel? I think in our CICD it was always sequential? since this stop LR server pattern is also used in multiple LR ITs before, it's using the port number to find which LR server to shutdown

Reply:
why does this need a change in the current PR?